<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>路径导航</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>570</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="dist_text">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>960</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="result_text">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>960</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/build-School_navigator-Desktop_Qt_5_10_1_MinGW_32bit-Debug/campus.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ComplexSchool">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>综合学院楼</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExecuteBuilding">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>390</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>行政楼</string>
      </property>
     </widget>
     <widget class="QPushButton" name="InfoCenter">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>340</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>信息中心</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Library">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>210</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>图书馆</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TeachingBuilding">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>教学楼</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Court">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>270</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>篮球场</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Playground">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>运动场</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Stadium">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>180</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>体育馆</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Canteen">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>370</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>食堂</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StudentCenter">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>510</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>学生活动中心</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StudentDormAB">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>410</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>学生宿舍A,B座</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TeacherDepartment">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>450</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>教师公寓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CommerceStreet">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>330</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商业街</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StudentDormDEFH">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>280</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>学生宿舍D、E、F、H</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StudentDormC">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>340</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>学生宿舍C座</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StudentDormG">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>380</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>学生宿舍G座</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RestHouse">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>440</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>招待所</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ParkingLot">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>460</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>停车场</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>550</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="start">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>起点</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>运动场</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>体育馆</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>篮球场</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>教学楼</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>图书馆</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>信息中心</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>综合楼</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>行政楼</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学生宿舍D,E,F,H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学生宿舍C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>商业街</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>饭堂</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学生宿舍A,B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学生宿舍G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>教师公寓</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>招待所</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>停车场</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学生活动中心</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="end">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>终点</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_2">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>运动场</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>体育馆</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>篮球场</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>教学楼</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>图书馆</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>信息中心</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>综合楼</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>行政楼</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学生宿舍D,E,F,H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学生宿舍C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>商业街</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>饭堂</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学生宿舍A,B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学生宿舍G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>教师公寓</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>招待所</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>停车场</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学生活动中心</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>560</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>单击按钮将显示图片哟(＾Ｕ＾)ノ~ＹＯ</string>
      </property>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>tableWidget</zorder>
     <zorder>pushButton</zorder>
     <zorder>dist_text</zorder>
     <zorder>result_text</zorder>
     <zorder>ComplexSchool</zorder>
     <zorder>ExecuteBuilding</zorder>
     <zorder>InfoCenter</zorder>
     <zorder>Library</zorder>
     <zorder>TeachingBuilding</zorder>
     <zorder>Court</zorder>
     <zorder>Playground</zorder>
     <zorder>Stadium</zorder>
     <zorder>Canteen</zorder>
     <zorder>StudentCenter</zorder>
     <zorder>StudentDormAB</zorder>
     <zorder>TeacherDepartment</zorder>
     <zorder>CommerceStreet</zorder>
     <zorder>StudentDormDEFH</zorder>
     <zorder>StudentDormC</zorder>
     <zorder>StudentDormG</zorder>
     <zorder>RestHouse</zorder>
     <zorder>ParkingLot</zorder>
     <zorder>label_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>全路段导航</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查找所有景点路线</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="result_text_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>261</width>
        <height>560</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="campus_picture">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>531</width>
        <height>560</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/build-School_navigator-Desktop_Qt_5_10_1_MinGW_32bit-Debug/campus.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>运动场</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>体育馆</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>篮球场</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>教学楼</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>图书馆</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>信息中心</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>综合楼</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>行政楼</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>学生宿舍D,E,F,H</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>学生宿舍C</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>商业街</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>饭堂</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>学生宿舍A,B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>学生宿舍G</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>教师公寓</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>招待所</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>停车场</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>学生活动中心</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>选择起点</string>
      </property>
     </widget>
     <zorder>campus_picture</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>result_text_2</zorder>
     <zorder>comboBox_3</zorder>
     <zorder>label</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
